components/automate-ui/src/app/page-components/simple-line-graph/simple-line-graph.component.ts

```diff
@@ -229,21 +229,21 @@ export class SimpleLineGraphComponent implements OnChanges, OnInit {
       // add class to rotate labels when more than comfortable to fit in space
       .classed('turnt', () => this.xData.length > 7)
 
-      .on('mouseenter', () => {
-        this.handleHover(d3.event);
+      .on('mouseenter', (e) => {
+        this.handleHover(e);
       })
       .on('mouseout', () => {
         this.AllDeactivate();
       })
```

Comment:
I like the conciseness of event-feed-guitar-strings.component.ts::L392-399; consider making all these anonymous functions one-liners, too.